### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by mirii1994**
* Added the option to run metricbeat in debug mode by using the environment parameter "DEBUG" in the docker run command.
@shalper - can you please review the line I added to the parameters for all modules table & update in the docs?

### CODE REVIEW

**Comment by mirii1994** [comment]
@Doron-Bargo Do we want them to run in debug mode as default? I thought we would use it if clients have trouble in running it for the first time. It prints out A LOT of logs when in debug mode.

**Comment by Doron-Bargo** [comment]
Agree , that why I ask to add that line (like a warning) . Now notice that i wrote should instead of shouldn't

**Comment by mirii1994** [comment]
@Doron-Bargo Added. Let me know if it's ok :)

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ in the [_Modules_](#modules) section at the bottom of this doc.
 | LOGZIO_TYPE (Default: `docker-collector-metrics`) | This field is needed only if you're shipping metrics to Kibana and you want to override the default value. <br> In Kibana, this is shown in the `type` field. Logz.io applies parsing based on `type`. |
 | LOGZIO_LOG_LEVEL (Default: `"INFO"`) | The log level the module startup scripts will generate. |
 | LOGZIO_EXTRA_DIMENSIONS | Semicolon-separated list of dimensions to be included with your metrics (formatted as `dimensionName1=value1;dimensionName2=value2`). <br> To use an environment variable as a value, format as `dimensionName=$ENV_VAR_NAME`. Environment variables must be the only value in the field. If an environment variable can't be resolved, the field is omitted. |
+| DEBUG (Default: `false`) | Set to `true` if you want metricbeat to run in debug mode.<br/> **Note:** This option will generate quite a lot of debugging output, so you should use it if an error occurs while running the docker-collector in production.  |
```
Comment (shalper) [inline README.md:54]:
```suggestion
| DEBUG (Default: `false`) | Set to `true` if you want Metricbeat to run in debug mode.<br/> **Note:** Debug mode tends to generate a lot of debugging output, so you should probably enable it temporarily only when an error occurs while running the docker-collector in production.  |

```

**Pull request merged**